myTinyURL/convert/convert.py

```diff
@@ -2,12 +2,14 @@
 from django.conf import settings
 from .models import URL
 import random, string
+import time
 
 class Convert():
     """core engine of url conversion
     """
 
     SHORT_ID_LENGTH = 6
+    LATEST_SHORT_ID = '000000'
```

Comment:
1. Read other Django projects source code and find how they store service layer status
2. If still cannot, then use class member variable as what you're doing now. But instead of make it start from 0, read it from db first.